src/coreclr/versionfallback/_version.h

```diff
@@ -0,0 +1,29 @@
+#ifndef VER_COMPANYNAME_STR
+#define VER_COMPANYNAME_STR         "Microsoft Corporation"
+#endif
+#ifndef VER_FILEDESCRIPTION_STR
+#define VER_FILEDESCRIPTION_STR     "empty"
+#endif
+#ifndef VER_INTERNALNAME_STR
+#define VER_INTERNALNAME_STR        VER_FILEDESCRIPTION_STR
+#endif
+#ifndef VER_ORIGINALFILENAME_STR
+#define VER_ORIGINALFILENAME_STR    VER_FILEDESCRIPTION_STR
+#endif
+#ifndef VER_PRODUCTNAME_STR
+#define VER_PRODUCTNAME_STR         "Microsoft\xae .NET Framework"
```

Comment:
```suggestion
#define VER_PRODUCTNAME_STR         "Microsoft\xae .NET"
```

Reply:
I'll put out a PR to Arcade to update these defaults as well (since this is the fallback file, we won't actually use it in official builds).